README.md

```diff
@@ -27,7 +27,9 @@ If this is your first time contributing to Mattermost, first read the [Mattermos
 
 The quickest way to begin is editing directly on GitHub on your fork of the Mattermost docs repo. Click the **Edit** icon on the top right corner of the page you want to edit in the Mattermost documentation.
 
-If this is the first time you're contributing, click **Edit the file in your fork of this project** (pencil icon) on the top right corner to create a fork.
+If this is the first time you're contributing, follow the steps: 
```

Comment:
Consider changing "follow the steps:" to "follow _these_ steps:"

Reply:
Sure, updated.